wsgi/prz-recipe-options.rst

```diff
@@ -23,30 +18,131 @@ The default is to listen on 0.0.0.0:8080.
     listen directive in wsgi.ini (`listen = 0.0.0.0:`)
     Waitress will open on a random non-privileged port in this case.
 
-threads (vs zserver-threads)
-----------------------------
+threads
+-------
 
 The new ``threads`` option specifies the number of worker threads for both WSGI and ZServer.
 The WSGI default is 4 threads since this is the waitress default.
 The ZServer default of two threads remains unchanged.
+``zserver-threads`` can still be used to specify the number of worker threads for ZServer but not waitress.
```

Comment:
Use consistent backticking throughout the page. Either

```
``ZServer`` and ``waitress``
or
ZServer and waitress
```

are both acceptable when used only as package names or "proper nouns", but double backticks are needed when used as an inline literal.

I think the main reason to use double backticks is in the case when the package's proper name is lower-cased so that it stands out from the regular plain text, but that's only my opinion. The docs team ought to clarify this usage, but we got bigger fish to fry and I doubt we would all agree what is "correct". It's a style thing.

Reply:
Not entirely sure, but hopefully this is resolved with 442880674e459b2532bd93951798dc7441640c10

Reply:
Wrong commit hash?

Reply:
yes sorry should be 09b2fbe69e5b52c3e6bd7396245dbf65d8aa7bc5

Reply:
LGTM.